<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WiiBaFuDialog</class>
 <widget class="QDialog" name="WiiBaFuDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../wiibafu.qrc">
    <normaloff>:/appicon/images/appicon.png</normaloff>:/appicon/images/appicon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Transfer">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_Directory">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Directory:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_Directory">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_Open">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_FilePath">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>File path:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_FilePath">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_OpenFile">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Open file</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_ImageFormat">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Image format:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBox_ImageFormat">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <item>
             <property name="text">
              <string>Wii Plain ISO (*.iso)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wii Compact ISO (*.ciso)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wii ISO Archive (*.wia)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wii Disc Format (*.wdf)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wii Backup File System Container (*.wbfs)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="comboBox_compressionDefaults">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fast</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Good</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Best</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Other</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_SplitSize">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Split size and optional unit factor 'kmgtpe' (base=1000] or 'KMGTPE' (base=1024)
Default value is 4G</string>
            </property>
            <property name="text">
             <string>4G</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_SplitSize">
            <property name="text">
             <string>Split size:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pushButton_Split">
            <property name="text">
             <string>Split</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_compressionMethod">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>44</height>
          </size>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Method:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_compressionMethod">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="currentIndex">
             <number>3</number>
            </property>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Purge</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bzip2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LZMA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LZMA2</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Level:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_compressionLevel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="currentIndex">
             <number>5</number>
            </property>
            <item>
             <property name="text">
              <string>0 (None)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1 (Fast)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5 (Good)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9 (Best)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>Factor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_compressionFactor">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="text">
             <string>20</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Edit">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>ID:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_ID">
            <property name="inputMask">
             <string notr="true">&gt;NNNNNN; </string>
            </property>
            <property name="maxLength">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4" colspan="3">
           <widget class="QLineEdit" name="lineEdit_Name">
            <property name="maxLength">
             <number>63</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Region:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="comboBox_Region">
            <item>
             <property name="text">
              <string>Auto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Japan</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>USA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Europe</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Korea</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>IOS:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QSpinBox" name="spinBox_HighIOS">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QComboBox" name="comboBox_IOS">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>High:Low</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>High-Low</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Low</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QSpinBox" name="spinBox_LowIOS">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Modify:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QComboBox" name="comboBox_Modify">
            <item>
             <property name="text">
              <string>Auto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disc</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Boot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ticket</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>TMD</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>WBFS</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>All</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Encoding mode:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="comboBox_EncodingMode">
            <item>
             <property name="text">
              <string>Auto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hash only</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Decrypt</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Encrypt</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sign</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="3" colspan="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Common key:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QComboBox" name="comboBox_CommonKey">
            <item>
             <property name="text">
              <string>Standard</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Korean</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Directory</tabstop>
  <tabstop>pushButton_Open</tabstop>
  <tabstop>lineEdit_FilePath</tabstop>
  <tabstop>pushButton_OpenFile</tabstop>
  <tabstop>comboBox_ImageFormat</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../wiibafu.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WiiBaFuDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WiiBaFuDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
